irods/manager/data_object_manager.py

```diff
@@ -64,7 +67,7 @@ def should_parallelize_transfer( self,
         # Example: $ export IRODS_VERSION_OVERRIDE="4,2,9" ;  python -m irods.parallel ...
         # ---
         # Delete the following line on resolution of https://github.com/irods/irods/issues/5932 :
-        if getattr(self.sess,'ticket__',None) is not None: return False
+#       if self.sess.ticket__: return False
```

Comment:
Are we waiting to merge this until after https://github.com/irods/irods/issues/5932 is resolved? Even if it is resolved, do we need to leave in this check for older versions of iRODS?

Reply:
I believe this is resolved, also for earlier iRODS versions.  Ticket API was doing dumb things, and getting in its own way. More details once I've whittled this PR down to minimal length.